src/components/RoarDataTable.vue

```diff
@@ -444,18 +444,19 @@ let dateFields = _filter(props.columns, (col) => _toUpper(col.dataType) === 'DAT
 dateFields = _map(dateFields, (col) => col.field);
 
 let toolTipByHeader = (header) => {
-  if (header === 'Word') {
-    return 'Assesses decoding skills at the word level. \n\n  Percentile ranges from 0-99 \n Raw Score ranges from 100-900';
-  } else if (header === 'Letter') {
-    return 'Assesses decoding skills at the word level. \n\n Percentile ranges from 0-99 \n Raw Score ranges from 0-90';
-  } else if (header === 'Phoneme') {
-    return 'Assesses phonological awareness: sound matching and elision. \n\n Percentile ranges from 0-99 \n Raw Score ranges from 0-57';
-  } else if (header === 'Sentence') {
-    return 'Assesses reading fluency at the sentence level. \n\n Percentile ranges from 0-99 \n Raw Score ranges from 0-130 ';
-  } else if (header === 'Palabra') {
-    return 'Assesses decoding skills at the word level in Spanish. This test is still in the research phase. \n\n  Percentile ranges from 0-99 \n Raw Score ranges from 100-900';
-  }
-  return '';
+  const headerToTooltipMap = {
+    Word: 'Assesses decoding skills at the word level. \n\n  Percentile ranges from 0-99 \n Raw Score ranges from 100-900',
+    'Letter Names and Sounds':
```

Comment:
I believe we've since reduced the header to just "Letter"
```suggestion
    Letter:
```